hardhat.config.js

```diff
@@ -56,26 +100,6 @@ task("deploy", "Deploy Colony Network as per truffle-fixture.js").setAction(asyn
   await deployNetwork();
 });
 
```

Comment:
Why remove the `coverage` task? Is it using the `test` task under-the-hood?

Reply:
It does ([solidity-coverage call](https://github.com/sc-forks/solidity-coverage/blob/master/plugins/hardhat.plugin.js#L271), [hardhat definition](https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-core/src/builtin-tasks/task-names.ts#L86)).